src/parsers/event_parser.py

```diff
@@ -8,17 +8,17 @@
 
 class EventParser():
 
-    def __init__(self, comparable_features):
-        '''Features found in imported comparable_features.json'''
-        self.comparable_features = comparable_features
+    def __init__(self):
```

Comment:
Features should be checked against the imported comparable features, not hardcoded